R/OTTER.R

```diff
@@ -0,0 +1,70 @@
+#' Run OTTER in R
+#' 
```

Comment:
@twangxxx here is an example of function documentation formatting. Would be grand to have stuff like this for the other functions.